### ISSUE TITLE
Unable to change the color of marks in the gutter

### ISSUE THREAD

**Issue by stevenxxiu** [NeoVintageous/NeoVintageous#982]
**Describe the bug**

I followed <https://github.com/NeoVintageous/NeoVintageous/blob/1.32.1/README.md#customize-markology-mark-colors> to change the color of marks in the gutter, but it does nothing.

**To Reproduce**

Steps to reproduce the behaviour:

1. Add to my color scheme:

```
{
    "scope": "neovintageous.mark",
    "foreground": "yellow",
},
```

2. Press `ma` to add a mark.

**Expected behaviour**

The mark in the gutter turns yellow.

**Actual behaviour**

The mark in the gutter stays cyan.

**Versions**

- *NeoVintageous*: v1.32.1
- *Sublime Text*: `startup, version: 4155 linux x64 channel: dev`

**Additional Information**

NA

**Comment by gerardroche** [comment]
I'm unabe to reproduce the issue.

Things to try:

- Restart Sublime
- Check the console for errors: `Menu > View > Show Console`
- Perhaps if you had previous marks they are restored on restart and don't have
  the new scope `neovintageous.mark`. Try deleting the mark and creating it
  again:

  ```
  :delmark a<CR>
  ma
  ```
- Try a hex color (though I don't know why that would be an issue):

  ```
  {
      "rules": [
          {
              "scope": "neovintageous.mark",
              "foreground": "#ff0000"
          }
      ]
  }
  ```

**Comment by gerardroche** [comment]
Sorry, the delete mark command is:

```
:delmarks a
```

**Comment by gerardroche** [comment]
What color scheme are you using?

Check your preferences "color_scheme" value.

Try a different color scheme. It's possible that some plugin like bracket
highilighter is caching an old version of your color scheme which is not
customizable. For instance, some users had this very issue with my own color
sceme MonokaiFree:

> I checked the color scheme select ui and actually I couldn't even select
> MonokaiFree correctly. It only listed a one MonokaiFree variant coming from
> User/Color Highlighter/themes/MonokaiFree.tmTheme (no idea how this came about)
>
> This seems like a UI limitation where it can only handle one Color Scheme with
> the same name
>
> I then set the color scheme directly in the settings "color_scheme":
> "MonokaiFree.sublime-color-scheme" and deleted the other User/Color
> Highlighter/themes/MonokaiFree.tmTheme, and now it works and the color scheme
> selector shows it correctly as selected.
>
> This was probably a resolution error on behalf of Sublime Text.
>
> https://github.com/gerardroche/sublime-monokai-free/issues/7#issuecomment-1598482396

**Comment by stevenxxiu** [comment]
Hi sorry for the trouble. I had an idea of what might have went wrong last night, and fixed it now.

Turns out I wasn't using my generic color scheme for *Markdown* files. I forgot that I had a custom syntax file for this, that redirected to `MarkdownEditor.sublime-color-scheme`.

To fix this, I just created a new `MarkdownEditor.sublime-color-scheme` file in my settings directory. Then I included the parts I wanted to override, being what I wrote in my first post.
